test/model/dataTypes/MemberVariable.test.ts

```diff
@@ -0,0 +1,186 @@
+import "jest-extended";
+import Element from "../../../src/model/modelInfo/Element";
+import ChoiceElement, {
+  ChoiceType,
+} from "../../../src/model/modelInfo/ChoiceElement";
+import ListElement from "../../../src/model/modelInfo/ListElement";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../../src/model/modelInfo/ListElement"](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)